tests/dragonfly/cluster_test.py

```diff
@@ -1773,6 +1773,53 @@ async def node1size0():
         assert str(i) == await nodes[1].client.get(f"{{key50}}:{i}")
 
 
+@pytest.mark.parametrize("type", ["list", "hash", "string", "set", "zset"])
+@dfly_args({"proactor_threads": 2, "cluster_mode": "yes"})
+@pytest.mark.asyncio
+async def test_cluster_migration_huge_list(df_factory: DflyInstanceFactory, type):
```

Comment:
rename test_cluster_migration_huge_list -> test_cluster_migration_huge_container
I see you test string data type above in params but its not clear to me what is 
debug populate 1 k 10000 RAND TYPE string ELEMENTS 1000
what is the affect of elements param here?

Reply:
also lets add streams to tested types

Reply:
> what is the affect of elements param here?

It doesn't have any effect, I thought it's a "free test" so why not, but I removed it

Reply:
> also lets add streams to tested types

I don't have support for breaking streams, nor does StaticSeeder nor debug populate support streams :|